diff -Naurp cobbler-2.1.0~bzr1881.orig//build_backup/Makefile.rej cobbler-2.1.0~bzr1881//build_backup/Makefile.rej
--- cobbler-2.1.0~bzr1881.orig//build_backup/Makefile.rej	1969-12-31 19:00:00.000000000 -0500
+++ cobbler-2.1.0~bzr1881//build_backup/Makefile.rej	2011-01-19 09:46:09.000000000 -0500
@@ -0,0 +1,28 @@
+--- build_backup/Makefile
++++ build_backup/Makefile
+@@ -4,6 +4,7 @@
+ 
+ prefix=devinstall
+ statepath=/tmp/cobbler_settings/$(prefix)
++www-user=www-data
+ 
+ all: clean build
+ 
+@@ -42,7 +43,7 @@
+ 
+ install: build manpage
+ 	python setup.py install -f
+-	chown -R apache /usr/share/cobbler/web
++	chown -R $(www-user) /usr/share/cobbler/web
+ 
+ debinstall: manpage
+ 	python setup.py install -f --root $(DESTDIR)
+@@ -77,7 +78,7 @@
+ 	cp $(statepath)/dhcp.template /etc/cobbler/dhcp.template
+ 	cp $(statepath)/rsync.template /etc/cobbler/rsync.template
+ 	find /var/lib/cobbler/triggers | xargs chmod +x
+-	chown -R apache /var/www/cobbler
++	chown -R $(www-user) /var/www/cobbler
+ 	chmod -R +x /var/www/cobbler/web
+ 	chmod -R +x /var/www/cobbler/svc
+ 	rm -rf $(statepath)
diff -Naurp cobbler-2.1.0~bzr1881.orig//setup.py cobbler-2.1.0~bzr1881//setup.py
--- cobbler-2.1.0~bzr1881.orig//setup.py	2011-01-19 09:43:25.000000000 -0500
+++ cobbler-2.1.0~bzr1881//setup.py	2011-01-19 09:46:09.964314734 -0500
@@ -128,7 +128,13 @@ if __name__ == "__main__":
     libpath     = "/var/lib/cobbler/"
     logpath     = "/var/log/"
 
-    webroot     = "/var/www/"
+    if os.path.exists("/etc/Suse-release"):
+        webroot = "/srv/www/"
+    elif os.path.exists("/etc/debian_version"):
+        webroot = "/usr/share/cobbler/webroot/"
+    else:
+        webroot     = "/var/www/"
+
     if os.path.exists("/etc/SuSE-release"):
         webconfig  = "/etc/apache2/conf.d"
     elif os.path.exists("/etc/debian_version"):
@@ -173,8 +179,6 @@ if __name__ == "__main__":
             ("/usr/sbin", ["scripts/tftpd.py"]),
 
             ("%s" % webconfig,              ["config/cobbler.conf"]),
-            ("%s" % webconfig,              ["config/cobbler_web.conf"]),
-            ("%s" % initpath,               ["config/cobblerd"]),
             ("%s" % docpath,                ["docs/*.gz"]),
             ("installer_templates",         ["installer_templates/*"]),
             ("%skickstarts" % libpath,      ["kickstarts/*"]),
