diff -Naurp cobbler-2.1.0~git20110113.orig//setup.py cobbler-2.1.0~git20110113/setup.py
--- cobbler-2.1.0~git20110113.orig//setup.py	2011-01-13 10:36:32.000000000 -0500
+++ cobbler-2.1.0~git20110113/setup.py	2011-01-13 11:00:13.492998114 -0500
@@ -128,7 +128,13 @@ if __name__ == "__main__":
     libpath     = "/var/lib/cobbler/"
     logpath     = "/var/log/"
 
-    webroot     = "/var/www/"
+    if os.path.exists("/etc/Suse-release"):
+        webroot = "/srv/www"
+    elif of.path.exists("/etc/debian_version"):
+        webroot = "/usr/share/cobbler/webroot"
+    else:
+        webroot     = "/var/www/"
+
     if os.path.exists("/etc/SuSE-release"):
         webconfig  = "/etc/apache2/conf.d"
     elif os.path.exists("/etc/debian_version"):
