diff -Naurp cobbler-2.1.0~git20110113.orig//Makefile cobbler-2.1.0~git20110113//Makefile
--- cobbler-2.1.0~git20110113.orig//Makefile	2011-01-13 11:24:51.982788404 -0500
+++ cobbler-2.1.0~git20110113//Makefile	2011-01-13 11:26:13.688161690 -0500
@@ -37,7 +37,7 @@ build:
 	python setup.py build -f
 
 install: build
-	python setup.py install -f
+	python setup.py install -f --install-layout=deb
 	chown -R $(www-user) /usr/share/cobbler/web
 
 debinstall:
