=== modified file 'docs/cobbler-register.pod'
--- a/docs/cobbler-register.pod	2009-04-21 22:26:09 +0000
+++ b/docs/cobbler-register.pod	2010-11-30 20:57:01 +0000
@@ -1,6 +1,6 @@
 =head1 NAME
 
-cobbler-register
+cobbler-register - create a cobbler system record
 
 =head1 SYNOPSIS
 

=== modified file 'docs/cobbler.pod'
--- a/docs/cobbler.pod	2010-04-15 14:17:57 +0000
+++ b/docs/cobbler.pod	2010-11-30 20:57:55 +0000
@@ -1,5 +1,7 @@
 =head1 NAME
 
+cobbler - a provisioning and update server
+
 cobbler is a provisioning (installation) and update server.  It supports deployments via PXE (network booting), virtualization (Xen, QEMU/KVM, or VMware), and re-installs of existing Linux systems.  The latter two features are enabled by usage of 'koan' on the remote system.  Update server features include yum mirroring and integration of those mirrors with kickstart.  Cobbler has a command line interface, Web UI, and extensive Python and XMLRPC APIs for integration with external scripts and applications.
 
 =head1 SYNOPSIS

=== modified file 'docs/koan.pod'
--- a/docs/koan.pod	2009-04-21 22:26:09 +0000
+++ b/docs/koan.pod	2010-11-30 20:58:11 +0000
@@ -1,6 +1,6 @@
 =head1 NAME
 
-koan
+koan - kickstart over a network, client side helper for cobbler
 
 =head1 SYNOPSIS
 

