=== modified file 'setup.py'
Index: ubuntu/setup.py
===================================================================
--- ubuntu.orig/setup.py	2010-12-16 13:00:32.000000000 -0800
+++ ubuntu/setup.py	2010-12-16 13:01:24.751700002 -0800
@@ -165,7 +165,6 @@
             # tftpd, hide in /usr/sbin
             ("/usr/sbin", ["scripts/tftpd.py"]),
 
-            ("%s" % initpath,               ["config/cobblerd"]),
             ("%s" % docpath,                ["docs/*.gz"]),
             ("installer_templates",         ["installer_templates/*"]),
             ("%skickstarts" % libpath,      ["kickstarts/*"]),
