=== modified file 'setup.py'
Index: cobbler/setup.py
===================================================================
--- cobbler.orig/setup.py	2010-12-09 16:09:59.000000000 -0800
+++ cobbler/setup.py	2010-12-09 16:15:41.631694002 -0800
@@ -168,7 +168,6 @@
 
             ("%s" % webconfig,              ["config/cobbler_web.conf"]),
             ("%s" % webconfig,              ["config/cobbler.conf"]),
-            ("%s" % initpath,               ["config/cobblerd"]),
             ("%s" % docpath,                ["docs/*.gz"]),
             ("installer_templates",         ["installer_templates/*"]),
             ("%skickstarts" % libpath,      ["kickstarts/*"]),
