Description: Dont use hardcoded init paths.
Author: Chuck Short <zulcss@ubuntu.com>
Upstream: No
diff -Naurp cobbler-2.1.0~bzr1881.orig//cobbler/modules/sync_post_restart_services.py cobbler-2.1.0~bzr1881//cobbler/modules/sync_post_restart_services.py
--- cobbler-2.1.0~bzr1881.orig//cobbler/modules/sync_post_restart_services.py	2011-01-19 10:40:24.000000000 -0500
+++ cobbler-2.1.0~bzr1881//cobbler/modules/sync_post_restart_services.py	2011-01-24 13:34:09.641998720 -0500
@@ -42,10 +42,10 @@ def run(api,args,logger):
                 if rc != 0:
                    logger.error("dhcpd -t failed")
                    return 1
-                rc = utils.subprocess_call(logger,"/etc/rc.d/init.d/dhcpd restart", shell=True)
+                rc = utils.subprocess_call(logger,"service dhcpd restart", shell=True)
         elif which_dhcp_module == "manage_dnsmasq":
             if restart_dhcp != "0":
-                rc = utils.subprocess_call(logger, "/etc/rc.d/init.d/dnsmasq restart")
+                rc = utils.subprocess_call(logger, "service dnsmasq restart")
                 has_restarted_dnsmasq = True
         else:
             logger.error("unknown DHCP engine: %s" % which_dhcp_module)
@@ -53,9 +53,9 @@ def run(api,args,logger):
 
     if manage_dns != "0" and restart_dns != "0":
         if which_dns_module == "manage_bind":
-            rc = utils.subprocess_call(logger, "/etc/rc.d/init.d/named restart", shell=True)
+            rc = utils.subprocess_call(logger, "service named restart", shell=True)
         elif which_dns_module == "manage_dnsmasq" and not has_restarted_dnsmasq:
-            rc = utils.subprocess_call(logger, "/etc/rc.d/init.d/dnsmasq restart", shell=True)
+            rc = utils.subprocess_call(logger, "service dnsmasq restart", shell=True)
         elif which_dns_module == "manage_dnsmasq" and has_restarted_dnsmasq:
             rc = 0
         else:
diff -Naurp cobbler-2.1.0~bzr1881.orig//koan/app.py cobbler-2.1.0~bzr1881//koan/app.py
--- cobbler-2.1.0~bzr1881.orig//koan/app.py	2011-01-19 10:40:24.000000000 -0500
+++ cobbler-2.1.0~bzr1881//koan/app.py	2011-01-24 13:34:48.468562668 -0500
@@ -585,7 +585,7 @@ class Koan:
                     raise InfoException("virt-image not present, downlevel virt-install package?")
 
             # for both virt types
-            if os.path.exists("/etc/rc.d/init.d/libvirtd"):
+            if os.path.exists("/etc/init.d/libvirtd"):
                 rc = sub_process.call("/sbin/service libvirtd status", stdout=None, shell=True)
                 if rc != 0:
                     # libvirt running?
