diff -Naurp cobbler-2.1.0~git20110113.orig//build_backup/Makefile cobbler-2.1.0~git20110113//build_backup/Makefile
--- cobbler-2.1.0~git20110113.orig//build_backup/Makefile	2011-01-13 10:36:31.000000000 -0500
+++ cobbler-2.1.0~git20110113//build_backup/Makefile	2011-01-13 10:47:24.906401218 -0500
@@ -6,6 +6,7 @@ prefix=devinstall
 statepath=/tmp/cobbler_settings/$(prefix)
 
 all: clean build
+www-user: www-data
 
 clean:
 	-rm -rf build rpm-build
@@ -42,7 +43,7 @@ build: manpage
 
 install: build manpage
 	python setup.py install -f
-	chown -R apache /usr/share/cobbler/web
+	chown -R $(www-user) /usr/share/cobbler/web
 
 debinstall: manpage
 	python setup.py install -f --root $(DESTDIR)
@@ -77,7 +78,7 @@ restorestate:
 	cp $(statepath)/dhcp.template /etc/cobbler/dhcp.template
 	cp $(statepath)/rsync.template /etc/cobbler/rsync.template
 	find /var/lib/cobbler/triggers | xargs chmod +x
-	chown -R apache /var/www/cobbler
+	chown -R $(www-user) /var/www/cobbler
 	chmod -R +x /var/www/cobbler/web
 	chmod -R +x /var/www/cobbler/svc
 	rm -rf $(statepath)
diff -Naurp cobbler-2.1.0~git20110113.orig//Makefile cobbler-2.1.0~git20110113//Makefile
--- cobbler-2.1.0~git20110113.orig//Makefile	2011-01-13 10:36:31.000000000 -0500
+++ cobbler-2.1.0~git20110113//Makefile	2011-01-13 10:46:43.687348648 -0500
@@ -6,6 +6,7 @@ prefix=devinstall
 statepath=/tmp/cobbler_settings/$(prefix)
 
 all: clean build
+www-user: www-data
 
 clean:
 	-rm -rf build rpm-build
@@ -37,7 +38,7 @@ build:
 
 install: build
 	python setup.py install -f
-	chown -R apache /usr/share/cobbler/web
+	chown -R $(www-user) /usr/share/cobbler/web
 
 debinstall:
 	python setup.py install -f --root $(DESTDIR)
@@ -72,7 +73,7 @@ restorestate:
 	cp $(statepath)/dhcp.template /etc/cobbler/dhcp.template
 	cp $(statepath)/rsync.template /etc/cobbler/rsync.template
 	find /var/lib/cobbler/triggers | xargs chmod +x
-	chown -R apache /var/www/cobbler
+	chown -R $(www-user) /var/www/cobbler
 	chmod -R +x /var/www/cobbler/web
 	chmod -R +x /var/www/cobbler/svc
 	rm -rf $(statepath)
