Description: Add "ubuntu" as detected distribution.
Author: Clint Byrum <clint@ubuntu.com>
Bug: https://fedorahosted.org/cobbler/ticket/617 

Index: cobbler/cobbler/utils.py
===================================================================
--- cobbler.orig/cobbler/utils.py	2010-12-09 13:12:17.431694001 -0800
+++ cobbler/cobbler/utils.py	2010-12-09 13:15:55.611694001 -0800
@@ -917,6 +917,14 @@
     Determines what distro we're running under.  
     """
     if os.path.exists("/etc/debian_version"):
+       try:
+           release = sub_process.check_output(("lsb_release","--id","--short")).rstrip()
+           if release == 'Ubuntu':
+              return "ubuntu"
+       except sub_process.CalledProcessError:
+           pass
+       except OSError:
+           pass
        return "debian"
     elif os.path.exists("/etc/SuSE-release"):
        return "suse"
@@ -954,6 +962,10 @@
       rest = parts[1]
       make = "debian"
       return (make, float(version))
+   elif check_dist() == "ubuntu":
+      version = sub_process.check_output(("lsb_release","--release","--short")).rstrip()
+      make = "ubuntu"
+      return (make, float(version))
    elif check_dist() == "suse":
       fd = open("/etc/SuSE-release")
       for line in fd.read().split("\n"):
