Index: ubuntu/config/cobbler.conf
===================================================================
--- ubuntu.orig/config/cobbler.conf	2010-12-14 11:39:23.235488000 -0800
+++ ubuntu/config/cobbler.conf	2010-12-16 13:02:19.711700001 -0800
@@ -1,18 +1,25 @@
 # This configuration file allows cobbler data
 # to be accessed over HTTP.
 
-AliasMatch ^/cblr(?!/svc/)(.*)?$ "/var/www/cobbler$1"
-AliasMatch ^/cobbler_track(.*)?$ "/var/www/cobbler$1"
-AliasMatch ^/cobbler(.*)?$ "/var/www/cobbler$1"
+AliasMatch ^/cblr(?!/svc/)(.*)?$ "/usr/share/cobbler/webroot/cobbler$1"
+AliasMatch ^/cobbler_track(.*)?$ "/usr/share/cobbler/webroot/cobbler$1"
+AliasMatch ^/cobbler(.*)?$ "/usr/share/cobbler/webroot/cobbler$1"
 
-WSGIScriptAliasMatch ^/cblr/svc/([^/]*) /var/www/cobbler/svc/services.py
+WSGIScriptAliasMatch ^/cblr/svc/([^/]*) /usr/share/cobbler/webroot/cobbler/svc/services.py
 
-<Directory "/var/www/cobbler">
+<Directory "/usr/share/cobbler/webroot/cobbler">
     Options Indexes FollowSymLinks
     Order allow,deny
     Allow from all
 </Directory>
 
+<IfModule !proxy_module>
+LoadModule proxy_module /usr/lib/apache2/modules/mod_proxy.so
+</IfModule>
+<IfModule !proxy_http_module>
+LoadModule proxy_http_module /usr/lib/apache2/modules/mod_proxy_http.so
+</IfModule>
+
 ProxyRequests off
 
 ProxyPass /cobbler_api http://localhost:25151/
@@ -24,7 +31,7 @@
 # and is visited at http://.../cobbler_web not this URL.
 # this is only a pointer to the new page.
 
-<Directory "/var/www/cobbler/web/">
+<Directory "/usr/share/cobbler/webroot/cobbler/web/">
     Options Indexes FollowSymLinks
     Order allow,deny
     Allow from all
Index: ubuntu/config/cobbler_web.conf
===================================================================
--- ubuntu.orig/config/cobbler_web.conf	2010-12-14 11:39:23.225488000 -0800
+++ ubuntu/config/cobbler_web.conf	2010-12-16 13:01:38.661700001 -0800
@@ -1,13 +1,6 @@
 # This configuration file enables the cobbler web
 # interface (django version)
 
-<VirtualHost *:80>
-
-# Do not log the requests generated from the event notification system
-SetEnvIf Request_URI ".*/op/events/user/.*" dontlog
-# Log only what remains
-CustomLog logs/access_log combined env=!dontlog
-
 <Location "/cobbler_web">
     SetHandler python-program
     PythonHandler django.core.handlers.modpython
@@ -21,7 +14,3 @@
     Require valid-user
     PythonAuthenHandler cobbler_web.views
 </Location>
-
-
-</VirtualHost>
-
