diff -Naurp cobbler.orig//cobbler/modules/manage_import_debian_ubuntu.py cobbler//cobbler/modules/manage_import_debian_ubuntu.py
--- cobbler.orig//cobbler/modules/manage_import_debian_ubuntu.py	2011-01-12 10:49:29.832365488 -0500
+++ cobbler//cobbler/modules/manage_import_debian_ubuntu.py	2011-01-12 11:21:35.294189788 -0500
@@ -361,9 +361,9 @@ class ImportDebianUbuntuManager:
                 self.logger.info("following symlink: %s" % fullname)
                 os.path.walk(fullname, self.distro_adder, distros_added)
 
-            if ( x.startswith("initrd") or x.startswith("ramdisk.image.gz") or x.startswith("vmkboot.gz") ) and x != "initrd.size":
+            if ( x.startswith("initrd.gz") ) and x != "initrd.size":
                 initrd = os.path.join(dirname,x)
-            if ( x.startswith("vmlinu") or x.startswith("kernel.img") or x.startswith("linux") or x.startswith("mboot.c32") ) and x.find("initrd") == -1:
+            if ( x.startswith("linux") ) and x.find("initrd") == -1:
                 kernel = os.path.join(dirname,x)
 
             # if we've collected a matching kernel and initrd pair, turn the in and add them to the list
