diff -Naurp ubuntu.orig//cobbler/action_check.py ubuntu//cobbler/action_check.py
--- ubuntu.orig//cobbler/action_check.py	2010-12-16 16:34:15.280106228 -0500
+++ ubuntu//cobbler/action_check.py	2010-12-16 21:24:34.648834718 -0500
@@ -98,6 +98,9 @@ class BootCheck:
        return status
 
    def check_for_ksvalidator(self, status):
+       if self.check_dist == 'debian' or self.check_dist == 'ubuntu':
+           return
+
        if not os.path.exists("/usr/bin/ksvalidator"):
            status.append("ksvalidator was not found, install pykickstart")
 
@@ -137,6 +140,8 @@ class BootCheck:
               status.append(_("since iptables may be running, ensure 69, 80, and %(xmlrpc)s are unblocked") % { "xmlrpc" : self.settings.xmlrpc_port })
 
    def check_yum(self,status):
+       if self.checked_dist == "debian" or self.checked_dist == "ubuntu":
+            return
        if not os.path.exists("/usr/bin/createrepo"):
            status.append(_("createrepo package is not installed, needed for cobbler import and cobbler reposync, install createrepo?"))
        if not os.path.exists("/usr/bin/reposync"):
@@ -150,6 +155,9 @@ class BootCheck:
                    status.append(_("yum-utils need to be at least version 1.1.17 for reposync -l, current version is %s") % yum_utils_ver )
 
    def check_debmirror(self,status):
+       if self.checked_dist == 'redhat' or self.checked_dist == 'suse':
+            return
+
        if not os.path.exists("/usr/bin/debmirror"):
            status.append(_("debmirror package is not installed, it will be required to manage debian deployments and repositories"))
        if os.path.exists("/etc/debmirror.conf"):
@@ -266,11 +274,14 @@ class BootCheck:
        """
        Check if Apache is installed.
        """
-       if self.checked_dist == "suse":
-           rc = utils.subprocess_get(self.logger,"httpd -v")
-       else:
-           rc = utils.subprocess_get(self.logger,"apache2 -v")
-       if rc.find("Server") != -1:
+       rc = 0
+       if self.checked_dist == 'debian' or self.checked_dist == 'ubuntu':
+            if not os.path.exists("/etc/default/apache2"):
+               rc = -1
+       elif self.checked_dist == 'redhat' or self.checked_dist == 'suse':
+            if not os.path.exists("/etc/httpd/conf"):
+                rc = -1
+       if rc == -1:
            status.append("Apache (httpd) is not installed and/or in path")
 
 
@@ -405,6 +416,9 @@ class BootCheck:
        """
        Check that rsync is enabled to autostart
        """
+       if self.checked_dist == 'debian' or self.checked_dist == 'ubuntu':
+          return
+
        if os.path.exists("/etc/xinetd.d/rsync"):
           f = open("/etc/xinetd.d/rsync")
           re_disable = re.compile(r'disable.*=.*yes')
