diff -Naurp ubuntu.orig/cobbler/action_import.py ubuntu//cobbler/action_import.py
--- ubuntu.orig/cobbler/action_import.py	2010-12-16 14:23:11.054173520 -0500
+++ ubuntu//cobbler/action_import.py	2010-12-16 14:24:11.354125250 -0500
@@ -1287,9 +1287,9 @@ class DebianImporter ( BaseImporter ) :
 
 class UbuntuImporter ( DebianImporter ) :
 
-   def __init__(self,logger,rootdir,pkgdir):
-       DebianImporter.__init__(self,logger,rootdir,pkgdir)
-       self.breed = "ubuntu"
+  def __init__(self,logger,rootdir,pkgdir):
+      DebianImporter.__init__(self,logger,rootdir,pkgdir)
+      self.breed = "ubuntu"
 
   def get_release_files(self):
        if not self.get_pkgdir():
